kitsune/wiki/tests/test_templates.py

```diff
@@ -1620,6 +1620,30 @@ def test_self_approve_with_revision_contributors(self):
         assert rev1.creator.username in label
         assert u.username not in label
 
+    def test_review_passed_revision(self):
+        """Verify that its not possible to review a revision older than the current revision"""
+        r1 = revision(is_approved=False, save=True)
+        r2 = revision(document=r1.document, is_approved=True)
+        r2.save()
```

Comment:
Don't do that, just don't assign it to a name. Remove the `r2 = ` apart of the line.